### ISSUE TITLE
[Feature Request]: Icon Colors

### ISSUE THREAD

**Issue by zkisaboss** [edelvarden/material-fox-updated#78]
### Description

Can you add a subtle tint to the toolbar icons in dark mode to match Chrome's color scheme?

Also, the following is unrelated, but I want to move my icon/searchbar closer to the top of my screen.
Is this something I can do in my custom.css file?
![image](https://github.com/user-attachments/assets/19446faa-2b27-4e2e-801d-36a6cee8fbb9)

### Additional info (Optional)

1.31.0
MacOS 15.0.1

**Comment by edelvarden** [comment]
Check it out in the latest version.

**Comment by zkisaboss** [comment]
Is there any way to change the color of these icons

![image](https://github.com/user-attachments/assets/13331e26-f4a7-45a2-a7a1-28e2ff7c851f)
![image](https://github.com/user-attachments/assets/d4c00983-247e-4850-b0f4-6fc37cddd904)

**Comment by edelvarden** [comment]
> Is there any way to change the color of these icons

Yes, try these variables: `--md-icon-color-primary` or `--md-icon-color-secondary`
